Tasks/XamariniOS/task.json

```diff
@@ -11,7 +11,7 @@
     "author": "Microsoft Corporation",
     "version": {
         "Major": 1,
-        "Minor": 126,
+        "Minor": 129,
```

Comment:
Need to bump the version in task.loc.json, too

Reply:
thanks